scalameta/quasiquotes/shared/src/main/scala-3/scala/meta/internal/quasiquotes/QuasiquoteType.scala

```diff
@@ -0,0 +1,19 @@
+package scala.meta
+
+private[meta] enum QuasiquoteType:
+  case Term // q""
```

Comment:
i thought `q""` covered any `Stat`, no? not just terms but also  definitions etc.